setup.cfg

```diff
@@ -1,5 +1,45 @@
 [metadata]
-description-file = README.md
+name = libioc
+version = attr: libioc.__version__
+description = A Python library to manage jails with ioc{age,cell}.
+long_description = file: README.md
+long_description_content_type = text/markdown
+url =  https://github.com/bsdci/libioc
+author = Stefan Grönke
+author_email = stefan@gronke.net
+python_requires = >=3.6
+platforms =
+    FreeBSD
+license = BSD 2-Clause License
+keywords =
+    FreeBSD
+    jail
+    ioc
+classifiers =
+    License :: OSI Approved :: BSD 2-Clause License
+    Operating System :: POSIX :: BSD :: FreeBSD
+    Programming Language :: Python :: 3 :: Only
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Typing :: Typed
+
+[options]
+zip_safe = False
+include_package_data = True
+packages = find:
+install_requires =
+	gitpython
+	freebsd_sysctl==0.0.7
+	jail==0.0.11
```

Comment:
this keeps us only one step away from the current setup
we should use ranges in libioc, and only in ioc use exact versions of the published (on ports) packages

Reply:
I agree that major version pinning is sufficient. Meanwhile the interfaces of both tools are stable enough to not break.

Reply:
time for a 1.0 release then